hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/TransferPrecompile.java

```diff
@@ -330,7 +392,13 @@ public long getMinimumFeeInTinybars(final Timestamp consensusTime) {
                                 PrecompilePricingUtils.GasCostType.TRANSFER_HBAR, consensusTime)
                         / 2;
         accumulatedCost += transferOp.transferWrapper().hbarTransfers().size() * hbarTxCost;
-
+        if (numLazyCreates > 0) {
```

Comment:
Should we charge additional gas for lazy creates without checking `isLazyCreationEnabled` flag? 

I think with the current implementation there is a possibility that a user tries to execute a transfer precompile with auto create attempts but with the feature flag turned off - then their request would be rejected but they would be charged with the gas needed for executing these auto creates. I'm not sure this is the expected behaviour.

Reply:
Great catch! Fixed now